README.md

```diff
@@ -24,11 +24,8 @@ Let’s walk through setting up a project that depends on DynamoDB from the SDK
 
 ```javascript
 const {
-  DynamoDBClient
-} = require("@aws-sdk/client-dynamodb-node/DynamoDBClient");
-const {
-  ListTablesCommand
-} = require("@aws-sdk/client-dynamodb-node/commands/ListTablesCommand");
+  DynamoDBClient, ListTablesCommand
```

Comment:
Prettier will move them to two different lines:
```suggestion
  DynamoDBClient,
  ListTablesCommand
```

Prettier playground link: https://bit.ly/2oKKm1L